lib/game.js

```diff
@@ -27,6 +27,9 @@ Game.prototype.update = function() {
   var lastbackgroundObject = this.backgroundObjects[this.backgroundObjects.length - 1];
   var firstObstacle = this.obstacles[0];
   var lastObstacle = this.obstacles[this.obstacles.length - 1];
+  var jeepney = this.jeepney;
+  var obstacles = this.obstacles;
+  var motorcycle = this.motorcycle;
```

Comment:
I think we can remove lines 31-32. Both of these variables are defined but not used.